tests/unit/genomics/test_runbigscape.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from nplinker.genomics import bigscape
+from .. import DATA_DIR
+
+
+@pytest.mark.parametrize("version", [1, 2])
+def test_run_bigscape(tmp_path, version):
+    """Test whether BiG-SCAPE runs at all using the --help command"""
+    result = bigscape.run_bigscape(
+        antismash_path=tmp_path,
+        output_path=tmp_path,
+        extra_params="--help",
+        version=version,
+    )
+
+    assert result is True
+
+
+@pytest.mark.parametrize("version", [1, 2])
```

Comment:
```suggestion
@pytest.mark.skipif(os.getenv('GITHUB_ACTIONS') == 'true', reason="The test is time-consuming on CI")
@pytest.mark.parametrize("version", [1, 2])
```
Need to add `import os`